flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamNode.java

```diff
@@ -63,6 +66,7 @@
 	private TypeSerializer<?> typeSerializerIn1;
 	private TypeSerializer<?> typeSerializerIn2;
 	private TypeSerializer<?> typeSerializerOut;
+	private Map<OutputTag<?>, TypeSerializer<?>> typeSerializerMap;
```

Comment:
This is not used anywhere in the code. Can it be removed, along with the `getTypeSerializerOut()` and `setTypeSerializerOut()`?